### ISSUE TITLE
ffmpeg framerate not detected similar to issue 79

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google-code-export/mythbox#124]
```
What steps will reproduce the problem?
1.select a video for playback
2.
3.

What is the expected output? What do you see instead?
The video will playback without error
I see an error stating that ffmpeg cannot detect the framerate and that comm 
skipping may not be accurate.

What version of the product are you using? On what operating system?
mythbox from hg version 165
xbmc-svn 34440
Arch Linux

Please provide any additional information below.
I read issue 79 but that fix seems to only work with Windows. I checked to 
ensure that my path to ffmpeg is correct.  I'm not sure where to look to solve 
this.  I will attach the mythbox.log
```

Original issue reported on code.google.com by `pkwalke...@gmail.com` on 9 Oct 2010 at 10:32

Attachments:
- [mythbox.log](https://storage.googleapis.com/google-code-attachments/mythbox/issue-124/comment-0/mythbox.log)

**Comment by GoogleCodeExporter** [comment]
```
Repost your log after enabling debug logging:

MythBox -> Settings -> Advanced -> Enable Debug Logging
```

Original comment by `semir.pa...@gmail.com` on 12 Oct 2010 at 1:32

**Comment by GoogleCodeExporter** [comment]
```
I am replacing my MythBackend's motherboard.  I will get this re-posted as soon 
as its up and running again, later this week.
```

Original comment by `pkwalke...@gmail.com` on 13 Oct 2010 at 8:48

**Comment by GoogleCodeExporter** [comment]
```
Here is the new mythbox.log with debug logging enabled
```

Original comment by `pkwalke...@gmail.com` on 16 Oct 2010 at 7:53

Attachments:
- [mythbox.log](https://storage.googleapis.com/google-code-attachments/mythbox/issue-124/comment-3/mythbox.log)

**Comment by GoogleCodeExporter** [comment]
```
Fixed in changeset e647f8789a
```

Original comment by `semir.pa...@gmail.com` on 17 Oct 2010 at 2:35
- Changed state: **Fixed**
